nif_points_clustering:
  ros__parameters:
    cluster_size_min : 10
    cluster_size_max : 2000
    max_cluster_distance : 0.5


